### DISCUSSION TITLE
Unable to either start existing Codespace or export changes to a branch

### DISCUSSION THREAD

**Discussion by williamcotton** [community/community#4182] [Codespaces] [answered]
Please tell me there are just some temporary outages related to Codespaces because I'm worried that I'll lose a good amount of work!

**Comment by williamcotton** [comment, marked as answer]
After a number of attempts I was able to export the changes to a branch.

I then destroyed my existing Codespaces container and created a new one.

**Reply by derekbekoe** [reply]
@williamcotton Thanks for reporting this. Codespaces had a temporary incident impacting a subset of customers that has since resolved. Let us know if you continue to see issues!

**Reply by jaulz** [reply]
@derekbekoe does something like this still occur on a regular basis? I tried CodeSpaces in the past and could only regain my uncommitted data after a colleague of you sent it to me manually. Today, I gave it another try and after half a day the VM is again unreachable and I cannot even export my changes via "Export changes to a branch" 😞
